addon/webextension/selector/documentMetadata.js

```diff
@@ -18,7 +18,7 @@ this.documentMetadata = (function() {
   function getOpenGraph() {
     let openGraph = {};
     // If you update this, also update _OPENGRAPH_PROPERTIES in shot.js:
-    let forceSingle = `title type url`.split(/\s+/g);
+    let forceSingle = `title type url`.split(/\s{1,225}/g);
```

Comment:
We should probably remove the regex here, just do `let forceSingle = "title type url".split(" ");`